javascript_es6/1.a.browser-echobot/README.md

```diff
@@ -0,0 +1,42 @@
+The example shows the use of the botbuilder-js SDK for the browser using the [BotFramework-WebChat](https://github.com/Microsoft/BotFramework-WebChat) and a custom [WebChatAdapter](/src/webChatAdapter.js). After running the bot, to see it in action, visit `http://localhost:8080`
+
+## To try this sample
+- Clone the repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+- In a terminal, navigate to javascript_es6\1.a.browser-echobot
+    ```bash
+    cd javascript_es6\1.a.browser-echobot
+    ```
+- Point to the myget feed 
+    ```bash
+    npm config set registry https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
+    ```
+- Install modules and start the bot
+    ```bash
+    npm i & npm start
+    ```
+- To reset registry, you can do
+    ```bash
+    npm config set registry https://registry.npmjs.org/
+    ```
+
+### Visual Stydio Code:
```

Comment:
>Stydio [](start = 11, length = 6)

Are these steps for a browser bot? They don't seem like it, but I could be mistaken.